usr/src/cmd/idmap/idmapd/idmap_lsa.c

```diff
@@ -83,7 +83,7 @@ lookup_lsa_by_sid(
 
 	(void) snprintf(sid, sizeof (sid), "%s-%u", sidprefix, rid);
 
-	rc = smb_lookup_sid(sid, &acct);
+	rc = smb_lookup_lsid(sid, &acct);
 	if (rc != 0) {
 		idmapdlog(LOG_ERR, "Error:  smb_lookup_sid failed.");
```

Comment:
Should error message be changed too?

Reply:
no

Reply:
Why not?

Reply:
The error message is informative as-is.  (We failed looking up a SID).  It need not be the actual function name called.  The user doesn't need to know about the function name.
If this changes at all, it should be to remove those two underscores.

Reply:
I think either we should update the function name so that it's correct, or as you suggest we should fix the message to be english text; e.g., `Error: SMB SID lookup failed`.  Having it be a function name that isn't the right function name is confusing.

Reply:
Agreed.  Fix that and you can add me to the list of reviewers.

Reply:
Updated error messages in the fix for 11039